### PR TITLE
DOCS-1404-Delete-version-OCP

### PR DESCRIPTION
**Pull request by bmckercher123**
Product Version(s):
All OSS versions in Docusaurus.

Issue:
https://tigera.atlassian.net/browse/DOCS-1404

Link to docs preview:
https://deploy-preview-738--tigera.netlify.app/calico/3.24/getting-started/kubernetes/openshift/installation

https://deploy-preview-738--tigera.netlify.app/calico/3.24/getting-started/kubernetes/openshift/requirements

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

File: calico/getting-started/kubernetes/openshift/installation.mdx [hunk 1]
```diff
@@ -25,7 +25,7 @@ to install {{prodname}}.
 
 - Ensure that you have a [RedHat account](https://cloud.redhat.com/). A RedHat account is required to obtain the pull secret necessary to provision an OpenShift cluster.
 
-- Ensure that you have installed the OpenShift installer **v4.3 or later** and OpenShift command line interface from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
+- Ensure that you have installed the OpenShift installer and OpenShift command line interface from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/aws/installer-provisioned).
```
Comment (sujeet-kr) [inline calico/getting-started/kubernetes/openshift/installation.mdx]: should we call out `supported Openshift installer and Openshift command line ...` since we are removing `4.3 or later` ?

File: calico/getting-started/kubernetes/openshift/installation.mdx [hunk 2]
```diff
@@ -25,7 +25,7 @@ to install {{prodname}}.
...
 
 - Ensure that you have [generated a local SSH private key](https://docs.openshift.com/container-platform/4.1/installing/installing_aws/installing-aws-default.html#ssh-agent-using_installing-aws-default) and have added it to your ssh-agent
```
Comment (sujeet-kr) [inline calico/getting-started/kubernetes/openshift/installation.mdx]: Perhaps we should direct to the latest version of this page. Current one seems to be for `4.1`

File: calico/getting-started/kubernetes/openshift/requirements.mdx [hunk 3]
```diff
@@ -8,12 +8,6 @@ description: Review the requirements for using OpenShift with Calico.
...
 ## OpenShift requirements
 
-{{prodname}} {{version}} supports:
-
-- OpenShift Container Platform 4.3+
-
-Refer to the OpenShift documentation for additional requirements.
-
-- [OpenShift Container Platform](https://docs.openshift.com/container-platform/4.3/welcome/index.html)
+{{prodname}} {{version}} supports the OpenShift Container Platform. See the [OpenShift Container Platform documentation](https://docs.openshift.com/)n for additional requirements.
```
Comment (sujeet-kr) [inline calico/getting-started/kubernetes/openshift/requirements.mdx]: Would be good reword this to call out `supported Openshift Container Platform`.

File: calico/getting-started/kubernetes/openshift/installation.mdx (hunk 1 above)
Comment (bmckercher123) [reply calico/getting-started/kubernetes/openshift/installation.mdx]: Yes, good one. Fixed.

File: calico/getting-started/kubernetes/openshift/installation.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico/getting-started/kubernetes/openshift/installation.mdx]: Removed all references to specific version. Made content generic.

File: calico/getting-started/kubernetes/openshift/requirements.mdx (hunk 3 above)
Comment (bmckercher123) [reply calico/getting-started/kubernetes/openshift/requirements.mdx]: Fixed.

**Review by ctauchen — APPROVED** [review]
LGTM, @bmckercher123
